applications/GeoMechanicsApplication/custom_utilities/math_utilities.h

```diff
@@ -7,1085 +7,22 @@
 //
 //  License:         geo_mechanics_application/license.txt
 //
-//  Main authors:    JMCarbonell,
-//                   Vahid Galavi
+//  Main authors:    Richard Faasse
 //
 
 #pragma once
 
-#ifdef FIND_MAX
-#undef FIND_MAX
-#endif
+#include <vector>
 
-#define FIND_MAX(a, b) ((a) > (b) ? (a) : (b))
-
-// System includes
-#include <cmath>
-
-// External includes
-
-// Project includes
-#include "geometries/point.h"
-#include "includes/global_variables.h"
-#include "utilities/math_utils.h"
+#include "includes/kratos_export_api.h"
+#include "includes/ublas_interface.h"
 
 namespace Kratos
 {
+
 class KRATOS_API(GEO_MECHANICS_APPLICATION) GeoMechanicsMathUtilities
```

Comment:
Is there need to have 'KRATOS_API(GEO_MECHANICS_APPLICATION)' now?

Reply:
I have tried to remove the export and rebuild the project, and that worked just fine. The export is just for changing the visibility of the function at the DLL level. As long as the function is being called by other functions that are in the same DLL, the export should not be needed. Only when functions that live in other DLLs need to access this function, the visibility becomes important (or else, linker errors will occur).

Reply:
Ahh okay, I think I was confused because in the past we needed to add the export when splitting into cpp and header for the processes, probably because they are also used in python then?

In any case it's necessary for gtest so indeed it doesn't really matter.